pennylane/ops/qubit/state_preparation.py

```diff
@@ -497,3 +570,48 @@ def circuit():
     """int: Number of trainable parameters that the operator depends on."""
 
     grad_method = None
+
+
+def _sparse_statevec_permute(state: csr_matrix, wires: list, wire_order: list) -> csr_matrix:
+    """Permutes the wires of a statevector represented as a scipy.sparse.csr_matrix.
```

Comment:
'Permute' is not accurate. What we implemented here are two features: embedding and permute. See the previous `StatePrep.state_vector` method for reference.

Reply:
That is a sensible observation. It would be nice to update the function docs to highlight this fact.